### PR TITLE
Updating glue-vispy-viewers

### PR DESCRIPTION
**Pull request by ryanskeith**
Updating from 0.12.2 to 1.0.5:
- Updating version, hash
- Updated dependencies

# Package Info
Source: https://github.com/glue-viz/glue-vispy-viewers/tree/v1.0.5
Diff: https://github.com/glue-viz/glue-vispy-viewers/compare/v0.12.2...v1.0.5

### CODE REVIEW

**Comment by ryanskeith** [comment]
Updated vispy to 0.12.1 and that made linux-aarch64 to build.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,31 +20,45 @@ requirements:
     - python
     - setuptools
     - setuptools_scm
+    - wheel
 
   run:
     - python
-    - numpy >=1.9
+    - numpy >=1.17
```
Comment (cbouss) [inline recipe/meta.yaml:27]: I don't see where the bound of numpy is set, is that really required?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -7,11 +7,11 @@ package:
 source:
   fn: glue-vispy-viewers-{{version}}.tar.gz
   url: https://pypi.io/packages/source/g/glue-vispy-viewers/glue-vispy-viewers-{{version}}.tar.gz
-  sha256: 3bc00472624e53c2d5ecb50d7bce29acdb2e6f6459694188044daa62c76ae8cf
+  sha256: 9d126735b1c1b08a56e02e11d0eba20c040ff934c8d1fd6644e7ee085a420569
 
 build:
-  noarch: python
   number: 0
+  skip: True  # [py<37 or ppc64le or s390x]
```
Comment (marcoesters) [inline recipe/meta.yaml:14]: This package is compatible with python 3.6: https://github.com/glue-viz/glue-vispy-viewers/blob/v1.0.5/setup.cfg#L23

Also, can you leave a comment about why we are not building for `ppc64le` and `s390x`?

File: recipe/meta.yaml (hunk 1 above)
Comment (ryanskeith) [reply recipe/meta.yaml]: I can't recall why I updated this.  I bet I just used glue core's requirement but that's not best practice.

**Pull request merged**